mempool/clist_mempool.go

```diff
@@ -310,6 +315,9 @@ func (mem *CListMempool) globalCb(req *abci.Request, res *abci.Response) {
 			}
 			mem.metrics.RecheckTimes.Add(1)
 			mem.resCbRecheck(req, res)
+
+		default:
+			panic("unexpected value of RequestCheckTx.type")
```

Comment:
This should never happen because the `Type` field is set elsewhere in the same module before the struct value is passed to the callback. Added the formatting just to be more informative.